docs/docker.md

```diff
@@ -20,25 +38,24 @@ These values will be used by both the database and the app. Now, run either:
 
 #### For Python 2.7
 
-~~~
+```
 $ docker-compose up --build
-~~~
+```
 
 #### For Python 3.7
 
 ##### Note: Python 3.7+ is not fully supported by the AGAGD Application. More information can be found in issue #112.
 
-~~~
-$ docker-compose -f docker-compose-python3.yml up --build
-~~~
+```
+$ docker-compose -f docker-compose.yml up --build
+```
```

Comment:
This need to be updated we don't need the `-f` flag as `docker-compose.yml` is default in docker, and we are using Python 3+.